charts/atlas-advanced/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: atlas-advanced
+description: This chart showcases Atlas Advanced (dedicated) deployment
+
+type: application
+
+version: 1.0.0
+
+keywords:
+  - mongodb
+  - database
+  - nosql
+home: https://github.com/mongodb/mongodb-atlas-kubernetes
+icon: https://camo.githubusercontent.com/d473c25e361fb2206f06bdcf24028e8d52bc30d1/68747470733a2f2f6d6f6e676f64622d6b756265726e657465732d6f70657261746f722e73332e616d617a6f6e6177732e636f6d2f696d672f4c6561662d466f7265737425343032782e706e67
+appVersion: "1.0.0"
```

Comment:
Is this accurate?

Reply:
That's the default value of the chart. I wonder if we want to stick it to the AKO version